### PR TITLE
[15.0][MIG] analytic_contract_value

### PR DESCRIPTION
**Pull request by DhavalTforgeFlow**

### CODE REVIEW

**Review by JuanyDForgeflow — APPROVED** [review]
Code review LGTM

File: analytic_contract_value/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# © 2015 Eficent - Jordi Ballester Alomar
+# © 2015 Serpent Consulting Services Pvt. Ltd. - Sudhir Arya
```
Comment (AaronHForgeFlow) [inline analytic_contract_value/__manifest__.py]:
```suggestion
# Copyright 2015-23 ForgeFlow SL
# Copyright 2015 Serpent Consulting Services Pvt. Ltd. - Sudhir Arya
```

**Review by AaronHForgeFlow — COMMENTED** [review]
LG! just please update copyright header in files.

File: analytic_contract_value/__manifest__.py (hunk 1 above)
Comment (JuanyDForgeflow) [reply analytic_contract_value/__manifest__.py]: @JuanyDForgeflow It's fixed now

**Review by AaronHForgeFlow — APPROVED** [review]

**Pull request merged**